staging/src/k8s.io/client-go/pkg/apis/clientauthentication/install/install.go

```diff
@@ -22,12 +22,16 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/client-go/pkg/apis/clientauthentication"
+	"k8s.io/client-go/pkg/apis/clientauthentication/v1"
 	"k8s.io/client-go/pkg/apis/clientauthentication/v1alpha1"
+	"k8s.io/client-go/pkg/apis/clientauthentication/v1beta1"
 )
 
 // Install registers the API group and adds types to a scheme
 func Install(scheme *runtime.Scheme) {
 	utilruntime.Must(clientauthentication.AddToScheme(scheme))
 	utilruntime.Must(v1alpha1.AddToScheme(scheme))
+	utilruntime.Must(v1beta1.AddToScheme(scheme))
+	utilruntime.Must(v1.AddToScheme(scheme))
 	utilruntime.Must(scheme.SetVersionPriority(v1alpha1.SchemeGroupVersion))
```

Comment:
This needs to be v1 -> v1beta1 -> v1alpha1

Reply:
not right now, but someday i'd like to learn why

3a8980d7e200e58633556fed22fda851a8da492c

Reply:
99% sure this does not matter today because this group has no REST APIs associated with it (i.e. you could delete the `SetVersionPriority` line and it would be fine).  But if we are going to have the line, it should be "correct."  IIRC this controls stuff like the preferred version in discovery and REST storage.

Reply:
Per 3a8980d I believe you misunderstood me here - the `AddToScheme` is not order sensitive, but the params for `SetVersionPriority` are:

```go
utilruntime.Must(scheme.SetVersionPriority(v1.SchemeGroupVersion, ..., v1alpha1.SchemeGroupVersion))
```

I suggest deleting the line since it will probably cause more confusion than it is worth.

Reply:
ah, yes i did misunderstand you. 8639fb7eba5 should fix this